docs/content/index.md

```diff
@@ -11,6 +11,13 @@ kube-bind is a prototype project that aims to provide better support for service
 - The service provider does not inject controllers/operators into the service consumer's cluster.
 - A single vendor-neutral, OpenSource agent per consumer cluster connects it with the requested services.
 
+## Key Features
+
+- **Catalog API**: Organize exported services using Collections (folders) and Modules (service definitions)
```

Comment:
```suggestion
- **Catalog API**: `APIServiceExportTemplate`s for users to generate `APIServiceExport`s from and `Collection`s to aggregate bind-able resources in a folder
```

Not super sure about the wording here. Also not sure if it should be called `Catalog API` - maybe this should just focus on the UI portion with the web interface and Collections and a separate point goes into the templates? Dunno.